omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -125,6 +130,7 @@
  'routes.lua',
  'dispatch_route.lua',
  'check_token_using_endpoint.lua',
+ 'check_token_using_endpoint_required_recipe.lua',
```

Comment:
Looks like you're right, and it's because that response is my favourite piece of html ever

    <h1>Are You Looking For the Chef Server?</h1>
    <p>Hello! It looks like you were trying to browse to your Chef Server but you haven't installed the Management Console.</p>

@ryancragun are you looking into this or do you want me to file a big somewhere? 😉